powder_reduction.py

```diff
@@ -1,37 +1,142 @@
+#!/usr/bin/env python
+# coding: utf-8
+import numpy as np
 import scipp as sc
 from scipp import Dim
+from scipp.neutron.diffraction import load_calibration
+import smooth_data
 
+LAMBDA_RANGE = (0.7, 10.35)
 
-def powder_reduction(data, calibration=None):
+
+def powder_reduction(sample='sample.nxs', calibration=None):
```

Comment:
Would this function also be valid for processing the vanadium run or are there different steps?

Reply:
The treatment related to Vanadium could easily be added. 
For WISH, it involveds loading the corresponding files, binning and then dividing the "detectors"' dataset by the vanadium one.

Reply:
My question is whether the vanadium is processed in the same way before this division, i.e., whether you can just use the existing `powder_reduction` function without changes to process the vanadium.

Reply:
Looking at reduce.py, the correction for Vanadium looks similar to some of the steps applied to the detectors' counts. But I do not have an event file for Vanadium to run a quick test.

Reply:
Can you use the (smaller) files from the original workflow (from the Mantid systemtest)?

Reply:
I managed to load a nexus file for Vanadium. But it is only related to one of the panels/groups (either 5 or 6).

Reply:
I will ask for a full one. In the meantime it would be good to understand whether there are additional/different steps in the processing when handling vanadium.

Reply:
One step which is not included in the current workflow is the subtraction of the `empty can` data.